cosmwasm/cosmwasm.nix

```diff
@@ -891,13 +907,54 @@ _: {
                           )"
                         )"
                   )
-
-                  # echo "deployments: $DEPLOYMENTS"
                 done <<< "$(echo "$DEPLOYMENTS" \
                   | jq -r '.[$key] | keys[]' \
                     --arg key "$key")"
             done
 
+            # get the ucs03 minter address and info
+            if [ "$(echo "$ADDRESSES" | jq '.app | has("ucs03")')" == "true" ]; then
+              MINTER_ADDRESS="$(
+                curl \
+                  --silent \
+                  '${rpc_url}/abci_query?path="/cosmwasm.wasm.v1.Query/SmartContractState"&data=0x'"$(
+                    buf \
+                      convert \
+                      ${cosmwasmProtoDefs}/cosmwasm.proto \
+                      --type=cosmwasm.QuerySmartContractStateRequest \
+                      --from=<(
+                        echo \
+                          "{\"address\":\"$(
+                            echo "$ADDRESSES" | jq -r '.app.ucs03'
+                          )\",\"query_data\":\"$(
+                            echo '{"get_minter":{}}' | base64 -w0
+                          )\"}"
+                      )#format=json \
+                      | xxd -c 0 -ps
+                    )" \
+                    | jq .result.response.value -r \
+                    | base64 -d \
+                    | buf \
+                      convert \
+                      ${cosmwasmProtoDefs}/cosmwasm.proto \
+                      --type=cosmwasm.QuerySmartContractStateResponse \
+                      --from=-#format=binpb \
+                    | jq .data -r \
+                    | base64 -d \
+                    | jq . -r
```

Comment:
No comments other than: I'm sorry lol

Reply:
this isn't even the worst part of this file